middleware/i18n/i18n.go

```diff
@@ -43,15 +43,20 @@ func (t *Translator) Load() error {
 	return t.Box.Walk(func(path string, f packr.File) error {
 		b, err := t.Box.MustBytes(path)
 		if err != nil {
-			log.Fatal(err)
-			return errors.WithStack(err)
+			wErr := errors.Wrapf(err, "Unable to read locale file %s", path)
```

Comment:
Error messages in Go are suppose to start with a lower case letter. I have no idea why. :)